src/client/static/hydration.tsx

```diff
@@ -11,10 +15,24 @@ export const hydrateApp = () => {
 
   const clientState = routingConfig.clientState;
 
+  const {
```

Comment:
What is this line doing ?

Reply:
extracting (destrucuring) the `mvtId` and `forcedTestVariants` from the client state. These values are from the the users cookie and query parameters respectively, which we pass through the server state to the client state.

Since the client state is never guaranteed we give them default values too

Reply:
I see, would a comment explaining that help? Also, how can you just reference `mvtId` later rather than `abTesting.mvtId` 🤯  ??

```
  hydrate(
  <ABProvider
      arrayOfTestObjects={tests}
      abTestSwitches={switches}
      pageIsSensitive={false}
      mvtMaxValue={1000000}
      mvtId={mvtId}
```

Reply:
Ah thanks for explaining what about 
```
const mvtId = clientState.abTesting?.mvtId || 0
const forcedTestVariants = clientState.abTesting?. forcedTestVariants || {}
```